prototype/{{cookiecutter.short_name}}/{{cookiecutter.short_name}}/translations/README.txt

```diff
@@ -0,0 +1,4 @@
+This translations directory is for .po|.mo files
+
+Visit XBlock Tutorial docs on Internationalization :
+https://github.com/edx/edx-documentation/blob/master/en_us/xblock-tutorial/source/edx_platform/edx_lms.rst
```

Comment:
Looks like you plan to change this to a readthedocs URL once it's ready, which is good.  I'd spell out ".po|.mo files", we don't need a cryptic notation. And while we're at it, "Visit the XBlock Tutorial docs on internationalization:"